nuxeo-java-client/src/main/java/org/nuxeo/client/objects/StringEntity.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * (C) Copyright 2020 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     jcarsique
+ *
+ */
+
+package org.nuxeo.client.objects;
+
+/**
+ * @since 3.8.1
+ */
+public class StringEntity extends Entity {
+
+    public StringEntity() {
+        super(EntityTypes.STRING);
+    }
+
+    public String getValue() {
+        return value;
+    }
+
+    public void setValue(String value) {
```

Comment:
What do you think about making it protected? 

IMHO, as this object only exists for an operation response we should discourage its usage for request since it won't work.